Pizza bytes

kathinka opened the pull request:
## Netlify link

https://pizzabytez.netlify.app/
https://pizzabytez.netlify.app/switch

kathinka committed:
first commit

kathinka committed:
added switch case

kathinka committed:
Update README.md

updated readme

AntonellaMorittu commented on code/index.html:
Please format your code to have the right spaces and indentation 👍

AntonellaMorittu commented on code/script.js:
```diff
@@ -1,19 +1,178 @@
 // Your code goes here
+
+function confirmOrder() {
+  let confirm = parseInt(prompt("You have ordered a " + age + " size " + food + ". \ncost: " + foodCost + " Bytecoin.\nPress 1 if this is correct. \nPress 2 if it's wrong."));
```
this is correct, but have you thought about using template literals here to improve readability? :)

kathinka replied:
good suggestion

AntonellaMorittu commented on code/switch.js:
Please format your code to have the right spaces and indentation 👍

AntonellaMorittu commented on code/switch.js:
```diff
@@ -0,0 +1,227 @@
+
+
+
+function ageCheck() {
+    let age = parseInt(prompt("How old are you?"));
+
+    if (age <= 0 || age > 124) {
+        alert("invalid");
+        ageCheck();
+    }
+
+    else if (age <= 5) {
+        alert(age + " years? better get your parents to order, you are way to young to order food on you own!");
+        ageCheck();
+    }
+    else if (age <= 15) {
+        order.push(orderType[0].childcost);
+        alert(age + " years? \n I guess you'll need a childrens portion. \n The total amount will be " + order[1] + " ByteCoin");
+        checkOut();
+    }
+    else if (age >= 16) {
+        order.push(orderType[0].cost);
+        alert("Seems like you are old enough for a regular portion.\n The cost is " + order[1] + " ByteCoin");
+        checkOut();
+    }
+    else {
+        alert("invalid");
+        ageCheck();
+    }
+}
```
this checks are very good, would have been nice to see it in action on your deployed version! 👍

kathinka replied:
hmm.. did you not see the provided links?
https://pizzabytez.netlify.app/
https://pizzabytez.netlify.app/switch

also after completion, you should see a button who will take you to the other example.

kathinka replied:
switched the scripts so now the switch versjon loads first.

AntonellaMorittu commented on code/switch.js:
```diff
@@ -0,0 +1,227 @@
+                break;
+            case 3:
+                orderType.push(menu[5])
+                order.push(menu[5].subtype)
+                alert("you chose " + order);
+                ageCheck();
+                break;
+            default:
+                alert("Sorry that's not on the  menu")
+                break;
+        }
+        break;
+
+    case 3:
+        let saladOption = parseInt(prompt("Press 1  🏛️🥬 for " + menu[6].subtype + " \n press 2 for 🇬🇷🥬 " + menu[7].subtype + " \n  press 3 for 🍉🥬  " + menu[8].subtype));
+        switch (saladOption) {
+            case 1:
+                orderType.push(menu[6])
+                order.push(menu[6].subtype)
+                alert("you chose " + order);
+                ageCheck();
+                break;
+            case 2:
+                orderType.push(menu[7])
+                order.push(menu[7].subtype)
+                alert("you chose " + order);
+                ageCheck();
+                break;
+            case 3:
+                orderType.push(menu[8])
+                order.push(menu[8].subtype)
+                alert("you chose " + order);
+                ageCheck();
+                break;
+            default:
+                alert("Sorry that's not on the  menu")
+                break;
+        }
+        break;
+}
```
Make sure to use a formatter to have the correct spacing and indentation, and change the tab size settings from 4 to 2😄 you can see now there is a lot of empty spaces in the code that compromise the readability ✨

AntonellaMorittu reviewed (approved):
Hi Kathinka, good job this week 🚀 the code meets the requirements and stretch goals, that's great! I would have liked to see the switch.js script executed in your deployed version. I left some comments specifically on some parts, please take a look at them :) Thank you ⭐

kathinka committed:
fixed spaces and identation.
used template literals for improving readabillity
Switched, so user sees switchcase first, and then get the option to try the first "versjon" with if else logic.

kathinka committed:
minor fix

kathinka committed:
cleaned up file
